src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2935,7 +2935,8 @@ export class ResultsViewPageStore {
             ],
             invoke: ()=>{
                 const dqf = this.molecularProfileIdToDataQueryFilter.result![q.molecularProfileId];
-                if (dqf) {
+                // it's possible that sampleIds is empty
+                if (dqf && dqf.sampleIds && dqf.sampleIds.length) {
```

Comment:
this change was reverted.  i think if you rebase it will go away.   (fyi i will be making a similar change shortly)

Reply:
So what should i do?

Reply:
Done @alisman. Check now?